backend/lcfs/web/api/dashboard/repo.py

```diff
@@ -0,0 +1,43 @@
+from logging import getLogger
+from fastapi import Depends
+from sqlalchemy import select, func, union_all, join, literal
+from sqlalchemy.ext.asyncio import AsyncSession
+from lcfs.db.dependencies import get_async_db_session
+from lcfs.db.models.transfer.Transfer import Transfer
+from lcfs.db.models.transfer.TransferStatus import TransferStatus, TransferStatusEnum
+from lcfs.db.models.compliance.ComplianceReport import ComplianceReport
+from lcfs.db.models.compliance.ComplianceReportStatus import ComplianceReportStatus, ComplianceReportStatusEnum
+from lcfs.db.models.initiative_agreement.InitiativeAgreement import InitiativeAgreement
+from lcfs.db.models.initiative_agreement.InitiativeAgreementStatus import InitiativeAgreementStatus, InitiativeAgreementStatusEnum
+from lcfs.db.models.admin_adjustment.AdminAdjustment import AdminAdjustment
+from lcfs.db.models.admin_adjustment.AdminAdjustmentStatus import AdminAdjustmentStatus, AdminAdjustmentStatusEnum
+from lcfs.web.core.decorators import repo_handler
+
+logger = getLogger("dashboard_repo")
+
+class DashboardRepository:
+    def __init__(self, db: AsyncSession = Depends(get_async_db_session)):
+        self.db = db
+
+    @repo_handler
+    async def get_director_review_counts(self):
```

Comment:
Your approach of getting all counts with one API call is better than mine; it reduces the number of API calls. 👍